cmd/internal/ecs/add.go

```diff
@@ -664,5 +715,42 @@ func fillFromFlags(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowSta
 }
 
 func addSecretState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowState], err error) {
-	return awf_error(errors.New("Unimplemented!"))
+	reportStep("Add AWS Secret")
+
+	key, secret := cfg.GetAPIKeyAndSecret()
```

Comment:
Would be nice if we requested a new key and secret from the back end for use in ECS. Not a blocker, though.